application-default.properties

```diff
@@ -448,13 +449,25 @@ mosip.digitalcard.pdf.password.enable.flag=true
 # Comma separated values of allowed auth types
 auth.types.allowed=otp-email,otp-phone,demo,bio-FINGER,bio-IRIS,bio-FACE,otp
 
-mosip.access_token.subject.claim-name=sub
-
-# It is used as a suffix for creating credential request ID using the RID.
-mosip.registration.processor.rid.delimiter=-PDF
-
 # Web UI Idle timeout related properties
 mosip.webui.auto.logout.idle=180
 mosip.webui.auto.logout.ping=30
 mosip.webui.auto.logout.timeout=60
 
+mosip.access_token.subject.claim-name=sub
+
+# It is used as a suffix for creating credential request ID using the RID.
+mosip.registration.processor.rid.delimiter=.pdf
```

Comment:
Previously value was -PDF and now it is changed to .pdf. Do we still need this property @Khuddusshariff0022 @dhanendra06

Reply:
Done